.processes/release.md

```diff
@@ -128,78 +128,89 @@ particular branch to deploy on every change.
+```
+
+7. On the `release/${RELEASE_NAME}` branch, check that everything is ready and push it to GitHub by executing : `git push origin`. Wait until the [deployment of the cluster](https://github.com/hoprnet/hoprnet/actions/workflows/deploy.yaml) has finished successfully.
+8. Create a Pull Request for tracking the release changes against the `master` branch. Remark in the PR description that it should never be merged!. Also use the label `DO NOT MERGE` and `release`. As a reference take a look at https://github.com/hoprnet/hoprnet/pull/4311
+9. On the `release/${RELEASE_NAME}` branch, create a topology cluster using the [script](./release.md#topology-deployment-script) mentioned at the end of this document.
+10. Create a release testnet page in the wiki at: https://www.notion.so/Testnets-e53255f7003f4c8eae2f1b6644a676e0
+    You may use previous testnet pages as templates. Ensure all started nodes are documented.
+11. Share the links to the release tracking issue, tracking PR and testnet wiki page in the `#release` Element channel.
     On the `#testing` channel, members are expected to run their own nodes (either AVADO or via their workstation) to participate in the release.
-11. Patches to the release are created via `hotfix/RELEASE_NAME/**` branches.
+12. Patches to the release are created via `hotfix/RELEASE_NAME/**` branches.
     Each of these merges will trigger a new release version, and re-build our infrastructure
     for that version. Upon successfully testing a release
-12. Once the first release version has been built and is running, the release branch should be merged-back into `master` once to trigger version upgrades on `master`. See the next section for details.
+13. Once the first release version has been built and is running, the release branch should be merged-back into `master` once to trigger version upgrades on `master`. See [the next](./release.md#release-merge-back) section for details.
 
 Once the release testing has concluded, or if any significant amount of patches has been applied to the release branch, the release branch should be merged back into `master` again.
 
 #### Release merge-back
 
-1. (on `release/${RELEASE_NAME}`) Create a PR branch off of the release branch: `git checkout -b merge-back-release-${RELEASE_NAME}`
-2. (on `merge-back-release-${RELEASE_NAME}`) Merge `master` into the branch: `git merge master`.
+1. On the `master` branch update the latest changes by executing: `git pull`
+2. On the `release/${RELEASE_NAME}` branch, create a PR branch of the release branch: `git checkout -b merge-back-release-${RELEASE_NAME}`
+3. On the `merge-back-release-${RELEASE_NAME}` branch, merge `master` into the branch: `git merge master`.
    For an example of a merge-back, take a look at older releases: https://github.com/hoprnet/hoprnet/pull/2956
    In case of conflicts (which is expected) changes from `master` have preference in the following cases:
```

Comment:
On first merge-back, the package version (in `package.json` files) from the new release is preferred. On subsequent merge-backs, the package version from `master` (with `-next.X` suffix) is preffered.

Reply:
Updated documentation